tools/lsp/ui/components/property-widgets.slint

```diff
@@ -295,6 +295,9 @@ component FloatWidget inherits VerticalLayout {
 }
 
 component StringWidget {
+    // FIXME: @ogoffart says the plural support is not working at this time, so
+    //        we do not offer it in the UI for the time being...
```

Comment:
I don't think this comment is necessary. 

Removing it was just a suggestion. Making it work would be better. I thought it may be easier to remove it. But keeping it broken doesn't make it good.